### PR TITLE
Mistake on using foreign key attribute

### LINKED ISSUE #10: Mistake on using ForeignKey attribute
You have used `ForeignKey` attribute in wrong way in all of your `Entities`. Example -

```csharp
        [ForeignKey("AssetId")]
        public int AssetId { get; set; }
        public virtual Asset Asset { get; set; }
```

which should be -

```csharp
        [ForeignKey("Asset")]
        public int AssetId { get; set; }

        public virtual Asset Asset { get; set; }
```

For more information check [Creating a More Complex Data Model for an ASP.NET MVC Application](https://docs.microsoft.com/en-us/aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/creating-a-more-complex-data-model-for-an-asp-net-mvc-application)'s `ForeignKey` attribute use

### PR DESCRIPTION
**Pull request by kajal-alpha**
Hi @mahedee bhai & @ratanparai ,
i have fix bug Mistake on using ForeignKey attribute. close #10 . Please check and give review.

### CODE REVIEW

File: src/AssetManager.Core/Entities/CustomFieldCustomFieldset.cs [hunk 1]
```diff
@@ -11,11 +11,11 @@ namespace AssetManager.Core.Entities
     public class CustomFieldCustomFieldset: Entity
     {
 
-        [ForeignKey("CustomFieldsId")]
+        [ForeignKey("CustomFields")]
         public int CustomFieldsId { get; set; }
         public virtual CustomFields CustomFields { get; set; }
 
-        [ForeignKey("CustomFieldsetId")]
+        [ForeignKey("CustomFields")]
```
Comment (ratanparai) [inline src/AssetManager.Core/Entities/CustomFieldCustomFieldset.cs]: This should be `CustomFieldSet`, the attribute name

File: src/AssetManager.Core/Entities/Location.cs [hunk 2]
```diff
@@ -15,7 +15,7 @@ public class Location: Entity
         public string Name { get; set; }
 
         [Display(Name = ("Parent"))]
-        [ForeignKey("ParentId")]
+        [ForeignKey("Location")]
```
Comment (ratanparai) [inline src/AssetManager.Core/Entities/Location.cs]: It should be `Locations`, the attribute name not the type

File: src/AssetManager.Core/Entities/Requests.cs [hunk 3]
```diff
@@ -11,14 +11,10 @@ namespace AssetManager.Core.Entities
...
     public class Requests: Entity
     {
-        [ForeignKey("AssetId")]
+        [ForeignKey("Asset")]
         public int AssetId { get; set; }
         public virtual Asset Asset { get; set; }
-
-        [ForeignKey("UserId")]
         public int UserId { get; set; }
-        public virtual User User { get; set; }
```
Comment (ratanparai) [inline src/AssetManager.Core/Entities/Requests.cs]: I think you have missed the `ForeignKey` attribute. Although Entity Framework is smart enough to understand the connection. 

You can also use 
```csharp
public int UserId { get ; set; }
[ForeignKey("UserId")]
public virtual User User { get; set; }
```

Comment (kajal-alpha) [reply src/AssetManager.Core/Entities/Requests.cs]: no need now. i will add this in the next work

Comment (ratanparai) [reply src/AssetManager.Core/Entities/Requests.cs]: @kajal-alpha : Okay.

**Pull request merged**